extensions/interactions/rules.json

```diff
@@ -0,0 +1,186 @@
+{
+  "CodeRepl": {
```

Comment:
Hm, I think we currently achieve the same thing (in this commit) using `karma-json-fixtures-preprocessor`. It allows the karma tests to read json files from the filesystem -- see my changes to `core/tests/karma.conf.js` and `extensions/interactions/rulesSpec.js` in this commit. If I'm not mistaken, static-module does basically the same thing, right?

Also, @maitbayev, FYI, the rulesSpec.js file contains the tests I was referring to in my reply -- they check that the code conforms to the single source of truth, which is intended to be the rules.json file.